eng/pipelines/coreclr/templates/perf-job.yml

```diff
@@ -165,7 +165,13 @@ jobs:
           artifactFileName: 'iOSSampleAppNoLLVM.tar.gz'
           artifactName: 'iOSSampleAppNoLLVM'
           displayName: 'iOS Sample App NoLLVM'
-
+      - template: /eng/pipelines/common/download-artifact-step.yml
+        parameters:
+          unpackFolder: $(Build.SourcesDirectory)/iosHelloWorld/llvm
+          cleanUnpackFolder: false
+          artifactFileName: 'iOSSampleAppLLVM.tar.gz'
+          artifactName: 'iOSSampleAppLLVM'
+          displayName: 'iOS Sample App LLVM'
```

Comment:
This should be conditioned on LLVM status we pass in, right? As it stands right now we will always download both artifacts.